event-sourcing/event-source-api/src/main/java/uk/gov/justice/services/eventsourcing/source/api/feed/common/PagingLinks.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.justice.services.eventsourcing.source.api.feed.common;
+
+import java.net.URL;
+
+public class PagingLinks {
+
+    private final URL previous;
```

Comment:
if some of these are optional should we use optionals ?

Reply:
previous and next can be optional, shall i change

Reply:
Changed to optionals and tests fixed